pkg/crypto/walletapi/wallet_api.go

```diff
@@ -56,6 +65,33 @@ type Crypto interface {
 	//
 	//		in case of error, the index will be -1
 	FindVerKey(candidateKeys []string) (int, error)
+
+	// ConvertToEncryptionKey will create and persist a Curve25519 encryption keypair
+	// created from the ed25519 keypair with the provided public key.
+	//
+	// Returns:
+	//
+	// []byte: The public key of the encryption keypair created. nil in case of error.
+	//
+	// error:
+	ConvertToEncryptionKey(pubkey []byte) ([]byte, error)
+
+	// AttachCryptoOperator attaches a crypto operator to this wallet, so the operator can use its private keys.
+	AttachCryptoOperator(cryptoOp KeyHolderUser) error
```

Comment:
same comment here about AttachCryptoOperator()

Reply:
This function is safe for external code to call - if a component needs (for example) to encrypt or sign something, and we've created a crypto operator that does that operation, then the component which needs the operation can call `myWallet.AttachCryptoOperator(operator)` to use the operator on their wallet's keys.